.github/CODEOWNERS

```diff
@@ -2,24 +2,27 @@
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence.
-@cbrunefearless @mackec1 @keithdadkins @christrotteradhoc @lsmitchell
+@cbrunefearless @mackec1 @keithdadkins @christrotteradhoc @lsmitchell @mjburling
 
 #BFD Pipeline and Server Codeowners
 /rfcs @cbrunefearless @mackec1 @keithdadkins @christrotteradhoc @lsmitchell
-/ops @keithdadkins @christrotteradhoc
+/ops @keithdadkins @christrotteradhoc @mjburling
 /apps @cbrunefearless @mackec1 @christrotteradhoc @lsmitchell
 
 #BFD Insights Codeowners
-/insights @keithdadkins @mackec1 @christrotteradhoc @lsmitchell
+/insights @keithdadkins @mackec1 @christrotteradhoc @lsmitchell @mjburling
 
 #BCDA Insights Codeowners
-/insights/terraform/projects/bcda @msnook @orenfromberg @keithdadkins
+/insights/terraform/projects/bcda @keithdadkins @mjburling
```

Comment:
could you add Trotter here?

Reply:
also to the other insights

Reply:
You bet. I think he's most everywhere now!